common/lib/xmodule/xmodule/templates/problem/checkboxes_response.yaml

```diff
@@ -23,6 +23,7 @@ metadata:
 
 data: |
       <problem>
+        <question>
         <p>Checkbox problems allow learners to select multiple options. Learners can see all the options along with the problem text.</p>
```

Comment:
Let's indent all of these templates properly. I can deal with an indent size of 2, but the hierarchy should be clear.